web/src/main/java/com/garret/movies/batch/MovieDbWriter.java

```diff
@@ -0,0 +1,33 @@
+package com.garret.movies.batch;
+
+import com.garret.movies.omdb.entity.OmdbMovie;
+import com.garret.movies.service.api.MovieService;
+import com.garret.movies.service.dto.response.MovieDto;
+import lombok.AllArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.TypeToken;
+import org.springframework.batch.core.configuration.annotation.StepScope;
+import org.springframework.batch.item.ItemWriter;
+import org.springframework.stereotype.Component;
+
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+@StepScope
+@AllArgsConstructor
+public class MovieDbWriter implements ItemWriter<OmdbMovie> {
+
+    private final MovieService movieService;
+    private ModelMapper modelMapper;
+
+    @Override
+    public void write(List<? extends OmdbMovie> list) throws Exception {
+        List<OmdbMovie> omdbMovieList = new ArrayList<>(list);
```

Comment:
@Garret-vin Why you need this?

Reply:
@Dominys I needed it earlier, but now i don't need it, so i fixed it